src/components/Widget/widget.tsx

```diff
@@ -146,10 +148,6 @@ export interface WidgetInterface extends BaseWidgetInterface {
   children?: React.ReactNode;
 }
 
-export const Widget: React.FC<
-  WidgetComponentInterface & WidgetInterface
-> = macroWrapper(WidgetComponent);
-
```

Comment:
Why has this been deleted?

Reply:
Does renaming WidgetComponent to Widget actually work? Surely it causes problems for non PV widgets which will require the baseWidget at runtime? Perhaps this is not causing errors from JSON...

Reply:
I think we can do some further cleaning up of these interfaces. We can talk about it next week.